src/js/models/geocoder/GeocoderSearch.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+define(
+  [
+    'underscore',
+    'backbone',
+    'models/geocoder/GoogleMapsGeocoder',
+    'models/geocoder/GoogleMapsAutocompleter',
+  ],
+  (
+    _,
+    Backbone,
+    GoogleMapsGeocoder,
+    GoogleMapsAutocompleter,
+    Prediction,
```

Comment:
`Prediction` is missing a dependency in the array and isn't used here (except in JSdocs)

Reply:
Done